### PR TITLE
Try new slack log link[full ci]

### PR DESCRIPTION
**Pull request by mhagen-vmware**
I had to remove the timestamp, so we can possibly over write logs if you restart a drone build, but you shouldn't do that anyways!

### CODE REVIEW

File: .drone.sec [hunk 1]
```diff
@@ -1 +1 @@
...
...JaaFqwsv6CVtWkF8x0JZwzShgtIs6RBF1Zgi3_j94QcKC_ih18QHxJ4jMMl448hrWz-F0u7Nxp4oVTltEwj1d5RSjPRlFIUpmNXNg-Irm4fXLYf0veHTchelU037l8Ow80XW1YlZDEl51B2NXbe0PXyURt_ItbXtpVKwJTO2jrVJsX15AHghty-uuvTscXkXQSm2UpwxDIMoYX90cl4KWNNLs4N_NgDDxnh-BSj_kBND_LKvHG_Jp69uwAQk79mDkbE2E9KOnzIQX5SjU8ZOJAjn9RMMH6fZkI9SNYFS0Aihmgk0Wd3PjHyKj9ykBZcBCO03e0-MY0nClgZLIx2jTjFJ5_zhbfH3oJRk6ylez0jvehyIk4wXX3fxU-KfZaOmwn7sNHu_6lFnajpzj6Xv-y5Pb7GVPg9_869l-mfcyue36DiqDN8gV40YyRcDSscqR7C3BdFgF0gb0FNxsYC3Gt-gvR6JBiKx82DsM_Jl_fDfBMN0YKi2gtozltY3bpj7Kv-m8JkBGRvfOuT-zFyWDY0EIxxPD4v_a5CjA05O11UaL2R7K_t5mY4a5mM8oHtx6C84JXF7bHmWEklim0qiTdfORxEcX9nvIyC3jfPMaGA5newqvy1_sRUWwYAeK0a7b4tv0yz4df_oJt-4BEOOtD11Zme9VVWBN6ZNOEu5Q63BJaJfcBssyTGJXy5PTGfoOmZsOFLuxHzsmNnwxXeoGgxyCC4KxJrd5raX-szhoBo2BO57zfpBNSqAbgWeJbWgPa8-bRpf1sn8PilNClwM2NBcyabUaJxPsoN4IU6GhgHVMAcrryBiJ5Yw8GgdiD97ZymY7IEh9HCm6X6ir7ng0l-ufIVyleljFlduZu98k8xX4GGbTHiE4WI8hv0ZbWz-QBieFE0Rfolzxq9CNsOk5pM6ske_BpAfbSeJv9cDq3HRSKhVYAdAL2sSFIuASWe8r2582nWi_bHOKsSG5WwXdoOgDacyF1iqgPvIoS_gnAKleaOib7LA79HNju3vywTCmaJcyfEol8qh9atVddgJdINNnatQdoxFJFbNVf2E3BUJSbWRqqRFQtsva-gCx_MOCC-kvqraYmCCASjtdX-kGpmcdgh5ZyTEKPy7TlcG2xI8qvJaKiFu2XkSWCau9zmcToDgoE5YXUi19FDGX92SJD6P4.jsW2HEkmLIDt4ab7aN90tg
```
Comment (jzt) [inline .drone.sec]: typo here:  `lFnajpzj6Xv-y5Pb7GVPg9_869l` ... that last part should be `8691` not `869l`.

Just kidding, 👍  LGTM 👍

**Comment by gigawhitlocks** [comment]
What's funny is I'm not sure why your original version didn't work (despite my teasing on Slack) now that I look at the source

That said, LGTM

**Comment by jzt** [comment]
LGTM

image

**Comment by gigawhitlocks** [comment]
eyyyy

**Comment by mdubya66** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**